.travis.yml

```diff
@@ -15,8 +16,9 @@ matrix:
 
 before_script:
   # --ignore-platform-reqs, because https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2722
-  - composer update -n --prefer-dist --ignore-platform-reqs
-  - composer require satooshi/php-coveralls dev-master --ignore-platform-reqs
+  - if [[ "${TRAVIS_PHP_VERSION}" != "7.0" ]]; then composer update -n --prefer-dist --ignore-platform-reqs; fi
+  - if [[ "${TRAVIS_PHP_VERSION}" == "7.0" ]]; then composer update -n --prefer-dist; fi
+  - composer require --dev php-coveralls/php-coveralls ^2.0.0 --ignore-platform-reqs
```

Comment:
maybe we should put this dependency in autoload-dev

Reply:
Either that or change it to the PHAR, [as Amp does it](https://github.com/amphp/amp/blob/216315ee5c59be6847af3391f11f04404d1ce659/.travis.yml).